js/the_hashtag_generator.js

```diff
@@ -0,0 +1,19 @@
+// https://www.codewars.com/kata/the-hashtag-generator/train/javascript
+
+function generateHashtag(str) {
+
+  const strWithoutSpaces = str.capitalize().replace(/\s/g, '');
+  if (strWithoutSpaces.length < 140 && strWithoutSpaces.length !== 0) {
+    return `#${strWithoutSpaces}`;
+  } else {
+    return false;
+  }
+}
+
+String.prototype.capitalize = function() {
```

Comment:
Are you aware of the problems that may occur by modifying the prototype of well-known objects, such as `String` :smile:? 

It's not wrong here, it's just a question

Reply:
As far I know the problem is that I could make some mess by editing an existing prototype function that can make some mess.

Reply:
Yup 😄 That's correct. If such a function will be added in the future, you will override it always.

That is why it's better to export functions than to add them to the prototype